ykman-gui/py/yubikey.py

```diff
@@ -132,6 +147,17 @@ def write_config(self, usb_applications, nfc_applications, lock_code):
             logger.error('Failed to write config', exc_info=e)
             return {'success': False, 'error': str(e)}
 
+    def refresh_piv(self):
+        with self._open_piv() as piv_controller:
+            return {
+                'has_protected_key': piv_controller.has_protected_key,
+                'pin_tries': piv_controller.get_pin_tries(),
+                'supported_touch_policies': [
```

Comment:
Not sure we want to implement pin and touch policies to begin with, they are not enabled by default in PIV manager.

Reply:
Good point, I'll remove the parts from here that aren't currently used.

Reply:
Done!